### PR TITLE
2022.3.0

### PR DESCRIPTION
**Pull request by frenck**

### CODE REVIEW

File: source/_posts/2022-03-02-release-20223.markdown [hunk 1]
```diff
@@ -0,0 +1,1020 @@
...
+The Updater integration has been deprecated; other integrations have replaced
+all its functionality. It is no longer a part of `default_config`,
+and if you have the `updater` integration defined directly in your YAML
+configuration, please remove it.
+
+- For the `binary_sensor.updater` entity, you can set up the
+  [Version integration](/integrations/version/) with the
+  "Home Assistant Website" source to get the same functionality back.
+- For the configuration options reporting and `include_used_components` for
+  the updater: configuration entry that was deprecated in 2021.4.
```
Comment (ludeeus) [inline source/_posts/2022-03-02-release-20223.markdown]:
```suggestion
- For the configuration options `reporting` and `include_used_components` for
  the `updater:` configuration entry that was deprecated in 2021.4.
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 2]
```diff
@@ -0,0 +1,1020 @@
...
+---
+
+<a href='/integrations/#version/2022.3'><img src='/images/blog/2022-03/social.png' style='border: 0;box-shadow: none;'></a>
+
+Did you know that today - March 2nd - in 1949, the first automatic street
+light was lit in New Milford, Connecticut, USA? Thirty-seven years later, we
+automate our entire homes 😎
+
+Anyways, back to 2022, Home Assistant Core 2022.3! And this release has
+a different and fresh "tune" to it! Yes, phun intended as this release brings
```
Comment (zsarnett) [inline source/_posts/2022-03-02-release-20223.markdown]:
```suggestion
a different and fresh "tune" to it! Yes, pun intended as this release brings
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 3]
```diff
@@ -0,0 +1,1020 @@
...
+tons of improvements involving media.
+
+And what is so cool about it? It is not just about browsing media,
+it is even more about using it! Using media allows me to make the automations
```
Comment (zsarnett) [inline source/_posts/2022-03-02-release-20223.markdown]:
```suggestion
it is even more about using it! Using media allows us to make the automations
```

Comment (frenck) [reply source/_posts/2022-03-02-release-20223.markdown]: This is an introduction signed by me, it is meant to be my perspective.

Comment (zsarnett) [reply source/_posts/2022-03-02-release-20223.markdown]: You say me and then our 

`Using media allows me to make the automation in our home`

Comment (frenck) [reply source/_posts/2022-03-02-release-20223.markdown]: our home... as in the home of me and my family.

Comment (zsarnett) [reply source/_posts/2022-03-02-release-20223.markdown]: Our seems like you are talking about the collective but maybe that's just how I am reading it. Could be both ways

Comment (frenck) [reply source/_posts/2022-03-02-release-20223.markdown]: I guess it doesn't harm to do this, can be read correctly in all ways in that case 👍 
Let adjust :)

**Review by zsarnett — DISMISSED** [review]

File: source/_posts/2022-03-02-release-20223.markdown [hunk 4]
```diff
@@ -0,0 +1,1019 @@
+---
+layout: post
+title: "2022.3: Select and play media"
+description: "Lots of media sources that can easily used with the Play media action!"
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:4]:
```suggestion
description: "Lots of media sources that can easily be used with the Play media action!"
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 5]
```diff
@@ -0,0 +1,1019 @@
...
+<img class="no-shadow" src='/images/blog/2022-03/mwc-vs-paper.png' alt='Screenshot showing the styling difference between MWC and paper style'>
+
+The screenshot above shows the new MWC styling on the left and the previous
+paper styling from an older Home Assistant version on the right. The styling
+change mainly impacts input fields look and feel.
+
+## Triggered!
+
+When working an automation, triggers now show live in the UI when they are
+triggering. Allowing you to validate you set it up correctly.
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:155]:
```suggestion
triggering, allowing you to validate you set it up correctly.
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 6]
```diff
@@ -0,0 +1,1019 @@
...
+
+In this little animation below, the remote is pressed a couple of times.
+The automation editor will show the "TRIGGERED" bar.
+
+<p class='img'>
+<img class="no-shadow" src='/images/blog/2022-03/triggered.gif' alt='Screen recording showing automations triggers triggering showing in the UI'>
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:161]:
```suggestion
<img class="no-shadow" src='/images/blog/2022-03/triggered.gif' alt='Screen recording showing automation triggers triggering showing in the UI'>
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 7]
```diff
@@ -0,0 +1,1019 @@
...
+Each time the trigger triggers it will be indicated in the UI.
+</p>
+
+Additionally, you can now also trigger each individual action in your automation
+to test if the action you are creating runs as expected:
+
+<img class="no-shadow" src='/images/blog/2022-03/run-action.png' alt='Screenshot showing how to run an individual from the automation editor'>
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:168]:
```suggestion
<img class="no-shadow" src='/images/blog/2022-03/run-action.png' alt='Screenshot showing how to run an individual action from the automation editor'>
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 8]
```diff
@@ -0,0 +1,1019 @@
...
+This breaking change does **NOT** apply to users of the [Z-Wave JS] integration.
+
+This is a breaking change notice regarding the older [Z-Wave] and [OpenZWave]
+integrations, which both have been previously deprecated.
+
+This release of Home Assistant Core 2022.3 will be the final release that
+provides these integrations. Both the old `zwave` and `ozw` integrations
+are pending removal for Home Assistant Core 2022.4.
+
+The `ozw` integration has been deprecated before as the upstream projects
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:352]:
```suggestion
The `ozw` integration has been deprecated before as the upstream
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 9]
```diff
@@ -0,0 +1,1019 @@
...
+([@corneyl] - [#66474]) ([documentation](/integrations/picnic))
+
+[@corneyl]: https://github.com/corneyl
+[#66474]: https://github.com/home-assistant/core/pull/6666474062
+
+{% enddetails %}
+
+{% details "Plugwise" %}
+
+Changing the scan interval options of Plugwise devices in the integration
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:632]:
```suggestion
Changing the scan interval option of Plugwise devices in the integration
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 10]
```diff
@@ -0,0 +1,1019 @@
...
+and if you have the `updater` integration defined directly in your YAML
+configuration, please remove it.
+
+- For the `binary_sensor.updater` entity, you can set up the
+  [Version integration](/integrations/version/) with the
+  "Home Assistant Website" source to get the same functionality back.
+- For the configuration options `reporting` and `include_used_components` for
+  the `updater:` configuration entry that was deprecated in 2021.4.
+  If you still want to share that,
+  please use the new [Analytics integration](/integrations/analytics/).
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:775]:
```suggestion
- The configuration options `reporting` and `include_used_components` for
  the `updater:` configuration entry was deprecated in 2021.4.
  If you still want to share that,
  please use the new [Analytics integration](/integrations/analytics/).
```

File: source/_posts/2022-03-02-release-20223.markdown [hunk 11]
```diff
@@ -0,0 +1,1019 @@
...
+
+[@ludeeus]: https://github.com/ludeeus
+[#67038]: https://github.com/home-assistant/core/pull/67038
+
+{% enddetails %}
+
+{% details "Vallox" %}
+
+The previously deprecated `vallox.set_profile` service has now been removed,
+please use the standard `fan.set_preset_mode` service as a replacement.
```
Comment (MartinHjelmare) [inline source/_posts/2022-03-02-release-20223.markdown:787]:
```suggestion
The previously deprecated `vallox.set_profile` service has now been removed.
Please use the standard `fan.set_preset_mode` service as a replacement.
```

**Review by zsarnett — DISMISSED** [review]

**Pull request merged**